src/components/NostoProvider.tsx

```diff
@@ -95,7 +95,7 @@ export default function NostoProvider(props: NostoProviderProps) {
       script.setAttribute("nosto-client-script", "")
 
       script.onload = () => {
-        if (typeof jest !== "undefined") {
+        if ("nostoReactTest" in window) {
```

Comment:
Is "nostoReactTest" part of window? Or is it a global variable?

Reply:
moved it properly to window